src/components/Gallery/index.scss

```diff
@@ -0,0 +1,41 @@
+.gallery {
+    .gallery-container {
+        width: 90%;
+        margin: 0 auto;
+
+        &__header {
+            display: flex;
+            align-items: center;
+            justify-content: space-between;
+            margin-bottom: 32px;
+        }
+
+        &__content {
+            display: flex;
+            overflow-x: auto;
+            margin: 36px 0 132px;
+        }
+    }
+}
+
+@media (max-width: 768px) {
```

Comment:
we put media query inside of every selector and pls use variables I created for media query to stay in tune